### PR TITLE
Msmtp-offline v1.0.

### PR DESCRIPTION
**Pull request by benjaminfrank**
msmtp wrapper which queues email sent when offline.

### CODE REVIEW

File: Library/Formula/msmtp-offline.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class MsmtpOffline < Formula
+  desc "msmtp wrapper which queues email sent when offline."
+  homepage "https://github.com/venkytv/msmtp-offline/"
+  url "https://github.com/venkytv/msmtp-offline/archive/v1.0.tar.gz"
+  sha256 "bc46754c74990da82f904a8b628d94e6e02d327bc515572f8c492adc40082041"
+  head "https://github.com/venkytv/msmtp-offline.git"
+
+  depends_on "msmtp"
+
```
Comment (bfontaine) [inline Library/Formula/msmtp-offline.rb]: Please remove the trailing spaces here.

File: Library/Formula/msmtp-offline.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  depends_on "msmtp"
+  
+  def install
+    bin.install "msmtp-offline"
+    bin.install "msmtp-queue"
+  end
+
+  test do
+  end
```
Comment (bfontaine) [inline Library/Formula/msmtp-offline.rb]: Please add a test in this block.

**Comment by bfontaine** [comment]
Audit is failing:

```
==> audit problems
msmtp-offline:
 * GitHub repository not notable enough (<10 forks, <10 watchers and <20 stars)
 * 9: Trailing whitespace was found
```

**Comment by benjaminfrank** [comment]
Removed the whitespace foo and wrote a proper test. Not sure about the "GitHub repository not notable enough" issue though.

**Comment by benjaminfrank** [comment]
Added a patch to support getting the account by email. PR submitted upstream.

**Comment by DomT4** [comment]
>  * GitHub repository not notable enough (<10 forks, <10 watchers and <20 stars)

This one is going to a fatal problem here for now. We require submissions to have some notability as otherwise they don't tend to get maintained, and a project being unpopular can = the project not being particularly active upstream.

Going to close here, but should the upstream project pass those minimum requirements please feel free to resubmit.

You could host it in a tap. There's some documentation on doing so [here](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/How-to-Create-and-Maintain-a-Tap.md). A tap provides the same experience as you'd expect with `brew` in regards to `update`, `upgrade`, `info` and so on.

**Pull request closed without merging**